source/citnames/source/Configuration.h

```diff
@@ -55,7 +55,7 @@ namespace cs {
     // overridden by command line arguments.
     struct Content {
         bool include_only_existing_source = false;
-        std::string duplicate_filter = DUPLICATE_FILENAME_OUTPUT;
+        std::string duplicate_filter_fields = DUPLICATE_INPUT_OUTPUT;
```

Comment:
I renamed the configuration parameter to `duplicate_filter_fields` that I think is a bit more clear than your suggestion.
Also I renamed the possible values to fit better with the name of the parameter.

Reply:
Yes, `duplicate_filter_fields` is a good name.

But I am still thinking about the `input` and `input_output` values. I think the `input` is confusing because the output JSON file has no field called input. It has `file` and `output`.

Reply:
So it's better to revert it to the old name

Reply:
Maybe call the values `all`, `file` and `file_and_output`?

Reply:
Didn't read your comment before pushing i named them `all`, `file` and `file_output`